content/asset-tracking/index.textile

```diff
@@ -0,0 +1,165 @@
+---
+title: Asset Tracking
+section: asset-tracking
+index: 0
+meta_description: "The Ably Asset Tracking solution provides a way to track multiple assets in realtime."
+meta_keywords: "Ably, Asset Tracking SDK, Mapbox, Mapbox Navigation SDK, Mapbox location enhancement, Ably realtime, map matching, location tracking, Publishing SDK, Subscribing SDK"
+languages:
+  - javascript
+  - android
+  - objc
+  - java
+  - swift
+jump_to:
+  Help with:
+    - Battery optimization scenarios#battery-optimization
+    - Resolution#resolution
+    - Resolution set#resolution-set
+    - Resolution constraints#resolution-constraints
+    - Resolution policy#resolution-policy
+    - See also#see-also
+---
+
+The Ably Asset Tracking solution provides a way to track the location of multiple assets in realtime. An asset is any object that moves geographically, and whose position needs to be monitored. The Ably Asset Tracking solution is powered by the Ably realtime network, the Ably Asset Tracking SDKs, and the Mapbox Navigation SDK with location enhancement.
+
+The Ably Asset Tracking solution provides two SDKs:
+
+* Publishing SDK - for embedding in apps on the asset to be tracked.
+* Subscribing SDK - for embedding in apps that want to observe the asset being tracked.
+
+As Ably is used as the underlying transport, you have direct access to your data and can use "Ably Integrations":https://ably.com/integrations for a wide range of applications, in addition to direct realtime subscriptions. Examples include: 
+
+* Passing data to a third-party service.
+* Persistence of data to a database for later retrieval.
+* Monitoring.
+
+As the asset being tracked moves, it publishes its position through the app to an Ably channel. Any subscribers to the channel can then receive positional information, which can be displayed on a detailed map.
+
+The following diagram provides an overview of an asset tracking use case:
+
+<a href="/images/screenshots/asset-tracking/overview.png" target="_blank">
+  <img src="/images/screenshots/asset-tracking/overview.png" style="width: 100%" alt="Overview">
+</a>
+
+In a typical asset tracking scenario the key participants are:
+
+- asset (trackable) := Item that the customer wants to track such as driver, vehicle, container, and so on. This also known as a @Trackable@ in the SDKs.
+- driver := Person driving the asset physically. Could also be a machine moving the asset under software direction, such as in the case of a delivery drone.
```

Comment:
In what way is this a "participant"? There's nothing in the SDKs that refer to this. I know that in the text that follows there are multiple references to "driver", but I'm not sure it qualifies for definition here. If it is defined here, I would prefer that it's not explicitly a "person".  We could define a more generic "carrier" term, maybe, and define that as the entity that is physically transporting the asset.